torchao/quantization/quant_api.py

```diff
@@ -120,9 +120,6 @@
     ZeroPointDomain,
 )
 from .subclass import (
```

Comment:
this file can be deleted as well?

Reply:
also probably https://github.com/pytorch/ao/blob/6cfa47705f60ea614695b52b4b120ac5fd84d1cb/torchao/quantization/dynamic_quant.py and https://github.com/pytorch/ao/blob/6cfa47705f60ea614695b52b4b120ac5fd84d1cb/torchao/quantization/weight_only.py

Reply:
Looks like classes from this file is still being used in autoquant. I don't think we can delete or even deprecate yet...